src/pages/LogView/LoadingPage/useResolveLogURLAndRenderingType.ts

```diff
@@ -169,13 +176,18 @@ export const useResolveLogURL = ({
         : constructEvergreenTaskLogURL(taskID, execution, origin, {
             text: false,
           });
+      renderingType = LogRenderingTypes.Default;
       break;
     }
     case LogTypes.EVERGREEN_TEST_LOGS: {
       if (!taskID || !execution || !testID || isLoadingTest) {
         break;
       }
-      const { url, urlRaw } = testData?.task?.tests.testResults[0]?.logs || {};
+      const {
+        renderingType: renderingTypeFromQuery,
+        url,
+        urlRaw,
+      } = testData?.task?.tests.testResults[0]?.logs || {};
```

Comment:
Can we leave a comment here. 
Something simple such as if we fetched test results use that URL otherwise fallback to hardcoded routes?